petition/index.html

```diff
@@ -5,10 +5,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>tbfighters.link</title>
     <meta name="description" content="Redirection server for TBfighters projects.">
-    <meta name="robots" content="noindex, nofollow">
-    <meta name="googlebot" content="noindex, nofollow">
+    <meta name="robots" content="noindex">
+    <meta name="googlebot" content="noindex">
     <meta property="og:title" content="tbfighters.link">
-    <meta property="og:description" content="Redirection server for TBfighters projects.">
+    <meta property="og:description" content="MSF Access Campaign">
```

Comment:
could you change this to something like MSF Access Campaign's Petition to lower Danaher's tests costs

Reply:
I was matching their og:title/description value.
![image](https://github.com/TBfighters/tblink/assets/9092153/7b1a1eaa-43ff-47a5-b399-5adab89d3903)

Reply:
ok, thanks for the information. Looks good then